### PR TITLE
git-clear: add force option

### PR DESCRIPTION
**Pull request by SimonTate**
Add a new -f/--force option to git-clear to allow usage in scripts,
or without user input.

Add -h/--help/? for help

Also update the manual for git-clear for these changes.

### CODE REVIEW

**Comment by SimonTate** [comment]
I've tested this on macOS only with bash 3.2.57. I've also run spellcheck, which didn't identify any new issues (only picked up read issue).

File: etc/git-extras-completion.zsh [hunk 1]
```diff
@@ -114,6 +114,11 @@ _git-changelog() {
         '(-l --list)'{-l,--list}'[list commits]' \
 }
 
+_git-clear() {
+    _arguments \
+        '(-f --force)'{-f,--force}'[force clear]' \
```
[the remark this answers is not in the document]
Comment (SimonTate) [reply etc/git-extras-completion.zsh:119]: Added in the latest commit

File: bin/git-clear [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+while [ "$1" != "" ]; do
+    case $1 in
+        -f|--force)
+            FORCE=1
+        ;;
+        -h|--help|?)
+            _usage
+            exit 1
+        ;;
+    esac
```
Comment (spacewander) [inline bin/git-clear:28]: Miss `shift`?

Comment (SimonTate) [reply bin/git-clear:28]: Apologies, yep. Added `shift` here in latest commit.

**Review by spacewander — APPROVED** [review]

**Pull request merged**